server/db/db.js

```diff
@@ -0,0 +1,12 @@
+const enviroment = process.env.NODE.ENV || 'development'
+const config = require('../knexfile.js')[enviroment]
+const connection = require('knex')(config)
+
+module.exports = {
```

Comment:
Please extract these functions out into `users.js` and pull in the db config from `db/index.js`

Reply:
This is complete